CSharpBasics2/CSharpBasics2/Program.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Xml.Linq;
+
+
+class Homework
+{
+    static void Main()
+    {
+       
+        {
+            // TASK_1
+
+            Console.WriteLine("Enter the number1:");
+            var numberToString = Console.ReadLine();
+            int q = Convert.ToInt32(numberToString);
+            double w = Convert.ToDouble(numberToString);
+            char r = Convert.ToChar(numberToString);
+            float t = Convert.ToSingle(numberToString);
+            Console.WriteLine($" {q}, {w}, {r}, {t}");
+            Console.ReadLine();
+
+        }
+
+        {
+            // TASK_2
+
+            Console.WriteLine("Enter the number2:");
+            var numberToString = Console.ReadLine();
+            int a = int.Parse(numberToString);
+            Console.WriteLine("Enter the number2:");
+            var numberToString1 = Console.ReadLine();
+            double b = double.Parse(numberToString);
+           
+
+        }
+
+        {
+            // TASK_3
+
+            Console.WriteLine("Enter the number3:");
+            var numberToString = Console.ReadLine();
+            int c;
+
+            var isNumber = int.TryParse(numberToString, out c);
+            Console.WriteLine($"Number is {c}");
+            Console.WriteLine("Success");
+
+        }
+
+        {
+            // TASK_4
+
+            Console.WriteLine("Enter the number4:");
+            var numberToString = Console.ReadLine();
+            int d;
+            var isNumber = int.TryParse(numberToString, out d);
+            int e = d + 3;
+            int f = e + d;
+            Console.WriteLine($" {d}, {e}, {f}");
+            Console.ReadLine();
+
+        }
+
+        {
+            // Task_5
+
+            string s1 = "Apples";

```

Comment:
do not forget about of number of each item :) this option is present in HW file :)